kotlinpoet/src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -40,6 +40,13 @@ public class AnnotationSpec private constructor(
   public val members: List<CodeBlock> = builder.members.toImmutableList()
   public val useSiteTarget: UseSiteTarget? = builder.useSiteTarget
 
+  /** Lazily-initialized toString of this type name.  */
+  private val cachedString by lazy {
```

Comment:
I copied the pattern in 

https://github.com/square/kotlinpoet/blob/master/kotlinpoet/src/main/java/com/squareup/kotlinpoet/TypeName.kt#L73

Reply:
nit but I think it's fine to make this an unsafe lazy -> `lazy(NONE)`

Reply:
Thanks! done [a39dbb7](https://github.com/square/kotlinpoet/pull/1506/commits/a39dbb7bd0107a27432b4aaa8c36bd175f0b0def)